test/test_context.py

```diff
@@ -498,3 +533,19 @@ def test_encode_geom(self):
         ewkb_resp = _encode_geom(geom)
         self.assertEqual(ewkb_resp, ewkb)
         self.assertIsNone(_encode_geom(None))
+
+    def test_dtypes2pg(self):
+        """context._dtypes2pg"""
+        from cartoframes.context import _dtypes2pg
+        results = {
```

Comment:
Are you going to throw something oddball at it?

Reply:
Good idea, I'll do that